informant.gemspec

```diff
@@ -11,4 +11,6 @@ Gem::Specification.new do |s|
   s.description = "Informant is a full-featured form builder for Ruby on Rails which promotes a simple syntax that keeps your views clean. Everything about a field (label, description, error display, etc) is encapsulated in a single method call."
   s.files       = `git ls-files`.split("\n") - %w[informant.gemspec Gemfile]
   s.require_paths = ["lib"]
+  s.add_dependency('actionview', '>= 4.0')
```

Comment:
Well, it's one thing to require changes to an app, it's another to drop compatibility entirely for a Rails version. Can you see if you can maintain support for 3.x? If it seems like it's going to be complicated we can skip it.